substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/UninterruptibleUtils.java

```diff
@@ -482,10 +481,9 @@ public static int modifiedUtf8Length(java.lang.String string, boolean addNullTer
          * interfere with the null terminator).
          */
         @Uninterruptible(reason = "Called from uninterruptible code.", mayBeInlined = true)
-        public static void toModifiedUtf8(java.lang.String string, Pointer buffer, Pointer bufferEnd, boolean addNullTerminator) {
+        public static void toModifiedUtf8(char[] string, Pointer buffer, Pointer bufferEnd, boolean addNullTerminator) {
```

Comment:
Is this dead code? If yes, I think we should remove it.

Reply:
Yes, it's dead code. This method was annotated with uninterruptable annotation, but implementation was wrong (String.length and String.charAt are interruptable methods, so I changed it). Maybe we can leave it as it is (with this small change) because it's a utils method and someone will maybe use it in the future.